esp-hal/MIGRATING-0.20.md

```diff
@@ -179,3 +179,22 @@ You can pass `NoPin` or `Level` as inputs, and `NoPin` as output if you don't ne
 -    .with_pins(Some(sclk), Some(mosi), NO_PIN, NO_PIN);
 +    .with_pins(sclk, mosi, Level::Low, NoPin);
```

Comment:
`Level::Low`?

Reply:
Oops, did I touch that in my PR?

Reply:
I don't think so, this looks fine.

Reply:
@plaes see a bit above

> You can pass `NoPin` or `Level` as inputs, and `NoPin` as output if you don't need a particular pin.

Though this needs to be updates because you can also pass `Level` when an output is expected if I remember my own changes correctly :D

Reply:
Oops, my bad... firstly I had missed that using level there is intended, and secondly it was the syntax highlighting of this "diff" in docs.